_includes/code/python/howto.configure.rbac.permissions.py

```diff
@@ -107,21 +107,21 @@ def reset_user(user: str, client: WeaviateClient):
 from weaviate.classes.rbac import Permissions
 
 permissions = [
-    Permissions.collections(
-        collection="TargetCollection*",
-        create_collection=True,
-        read_config=True,
-        update_config=True,
-        delete_collection=True,
-    ),
-    # Without the below permission, the user would not
-    # be able to create tenants in collections starting with "TargetCollection"
     Permissions.tenants(
         collection="TargetCollection*",  # Applies to all collections starting with "TargetCollection"
+        tenant="TargetTenant*",  # Applies to all tenants starting with "TargetTenant"
         create=True,  # Allow creating new tenants
         read=True,  # Allow reading tenant info/metadata
         update=True,  # Allow updating tenant states
-        delete=False,  # Dont't allow deleting tenants
+        delete=True,  # Dont't allow deleting tenants
```

Comment:
Hey @g-despot the comment and property here don't match

Reply:
Thanks for spotting this